posthog/async_migrations/examples/example.py

```diff
@@ -11,7 +13,6 @@
     PERSONS_DISTINCT_ID_TABLE_MV_SQL,
     PERSONS_DISTINCT_ID_TABLE_SQL,
 )
-from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
```

Comment:
yeah, there's a dusty corner of my brain that says `from posthog.settings blah` means you can't use settings override context manager in tests too